scripts/common.py

```diff
@@ -132,6 +133,16 @@ def ensure_release_scripts_folder_exists_and_is_up_to_date():
         subprocess.call(['git', 'pull', remote_alias])
 
 
+def require_gcloud_to_be_available():
+    status, msg = commands.getstatusoutput('gcloud')
+
+    if status == 32512:
+        # gcloud command does not exist.
+        raise Exception('%s.\n Please install gcloud '
```

Comment:
Is there a way to do this as part of the setup process (and restrict it to be installed in oppia_tools, like we currently do with the App Engine SDK)? That way, all of oppia's dependencies are not spread out all over the developer's system.